vela/repo.go

```diff
@@ -119,3 +119,19 @@ func (svc *RepoService) Chown(org, repo string) (*string, *Response, error) {
 
 	return v, resp, err
 }
+
+// Sync synchronizes a repo between the database and the SCM.
+func (svc *RepoService) Sync(org, repo string) (*string, *Response, error) {
```

Comment:
I think the pattern right now for the client services is to have the base part of the API path as its own service. So, I think this should be a new method under an `SCMService` and not under the `RepoService`

Reply:
Done!